### PR TITLE
[CORE-10715] Document IPForwarding Felix configuration param.

### PR DESCRIPTION
**Pull request by fasaxc**
Product Version(s):

OS v3.29.0
Enterprise v3.20.0-2.0, v3.19.3

Issue:

https://github.com/projectcalico/calico/pull/9313
https://github.com/projectcalico/calico/pull/9320
https://github.com/tigera/calico-private/pull/8100
https://github.com/tigera/calico-private/pull/8101
https://github.com/tigera/calico-private/pull/8102

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico/reference/felix/configuration.mdx [hunk 1]
```diff
@@ -98,7 +99,7 @@ The full list of parameters which can be set is as follows.
...
+| `mtuIfacePattern`                   | `FELIX_MTUIFACEPATTERN`                   | Pattern used to discover the host's interface for MTU auto-detection. [Default: ^((en&#124;wl&#124;ww&#124;sl&#124;ib)[opsvx]&#46;&#42;&#124;(eth&#124;wlan&#124;wwan)&#46;&#42;) ]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | regex                                                                                           |
```
Comment (ctauchen) [inline calico/reference/felix/configuration.mdx:102]: Did you intend to change something here? It's highlighted, but I can't see what's changed.

File: calico-enterprise/reference/component-resources/node/felix/configuration.mdx [hunk 2]
```diff
@@ -66,6 +66,7 @@ The full list of parameters which can be set is as follows.
...
+| `IPForwarding`                      | `FELIX_IPFORWARDING`                                                                                 | Controls whether Felix sets the host sysctls to enable IP forwarding.  IP forwarding is required   when using Calico for workload networking.  This should only be disabled on hosts where Calico is used for host protection.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | `Enabled` or `Disabled`                                                                         |
```
Comment (ctauchen) [inline calico-enterprise/reference/component-resources/node/felix/configuration.mdx]: There's a number of double spaces in this PR. Doesn't affect the output, but a single space is preferred!

Comment (ctauchen) [inline calico-enterprise/reference/component-resources/node/felix/configuration.mdx]:
```suggestion
| `IPForwarding`                      | `FELIX_IPFORWARDING`                                                                                 | Controls whether Felix sets the host sysctls to enable IP forwarding.  IP forwarding is required   when using Calico for workload networking.  This should be disabled only on hosts where Calico is used for host protection.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | `Enabled` or `Disabled`                                                                         |
```
Pedantic, but carries a different meaning.

**Review by ctauchen — COMMENTED** [review]
A few nits, otherwise LGTM. Feel free to merge with changes.

File: calico/reference/felix/configuration.mdx (hunk 1 above)
Comment (fasaxc) [reply calico/reference/felix/configuration.mdx:102]: There was a huge amount of whitespace at the end of that line.

File: calico-enterprise/reference/component-resources/node/felix/configuration.mdx (hunk 2 above)
Comment (fasaxc) [reply calico-enterprise/reference/component-resources/node/felix/configuration.mdx]: I still prefer double-spacing when working with fixed-width fonts but whatever.

**Comment by ctauchen** [comment]
LGTM, thanks! One last thing: If this goes in CE 3.19 +, does it also belong in OSS 3.27+? Currently it's included in 3.29 only.

**Comment by fasaxc** [comment]
No, I didn't backport it to v3.28/27, it was a customer ask to get it into v3.19.3.

**Pull request merged**